test/protocol/ConfigHandlerTest.js

```diff
@@ -1241,6 +1248,14 @@ describe("IBosonConfigHandler", function () {
           ethers.constants.AddressZero,
           "Invalid auth token contract address"
         );
+        expect(await configHandler.connect(rando).getAuthTokenContract(AuthTokenType.ENS)).to.equal(
+          ethers.constants.AddressZero,
+          "Invalid auth token contract address"
+        );
+        expect(await configHandler.connect(rando).getAuthTokenContract(AuthTokenType.Custom)).to.equal(
```

Comment:
Does it ever get here, it does seem like it would ever get here? Or perhaps I am mistaken....

Reply:
Calling `getAuthTokenContract(AuthTokenType.Custom)` should return the zero address, so it's a valid test case. If ` != AuthTokenType.Custom` were ever removed from the `require` in `setAuthTokenContract`, this test case would fail. That's what we want.